packages/ast-spec/src/base/BinaryExpressionBase.ts

```diff
@@ -1,8 +1,9 @@
+import type { PrivateIdentifier } from '../special/PrivateIdentifier/spec';
 import type { Expression } from '../unions/Expression';
 import type { BaseNode } from './BaseNode';
 
 export interface BinaryExpressionBase extends BaseNode {
```

Comment:
at this point we could probably just delete this interface entirely. Each node that extends this already declares its own `operator`, and with one node now declaring its own `left` - it's kind of no longer a "shared common base".